.github/scripts/s390x-ci/self-hosted-builder/helpers/app_token.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
On some OSes bash is in `/usr/bin/` rather than in `/bin`, this is why it's often prefixed with `/usr/bin/`